src/bluehawk/actions/snip.ts

```diff
@@ -122,6 +132,19 @@ export const formatInRst = async (
   return formattedSnippet;
 };
 
+export const formatInMd = (result: ProcessResult): string | undefined => {
+  const { document } = result;
+  const snippet = document.attributes["snippet"];
+  if (snippet === undefined) {
+    return undefined;
+  }
+  const language = document.extension.slice(1);
+  const startBackticks = `\`\`\`${language}\n\n`;
```

Comment:
This is one case where stylistically I'd prefer to see no template string literal:

```js
const start = ["```", language, "\n\n"].join("");
```